### ISSUE TITLE
GTP engine can't be synced to current state

### ISSUE THREAD

**Issue by DalurRain** [SabakiHQ/Sabaki#737]
Well, I see there is an issues with this problem showing “need more information”, I'm now provide some information here:

When I use Sabaki v0.33.4 whith Tencent‘s PhoenixGo-cpu-only engine, everything is OK, engine vs. engine just runs correctly, and then I update Sabaki version to v0.51.1, it gives me the error msg "GTP engine can't be synced to current state".
So from v0.33.4 to v0.51.1, I guess Sabaki has changed something(of course it changed very much) with loading engines? And what I should do to solve this problem? 

By the way, I use it on Windows 10

**Comment by yishn** [comment]
What's the other issue you're referring to? Why did you open a new issue for this then?

**Comment by yishn** [comment]
Please try with a GTP engine that I know is working with Sabaki, e.g. LeelaZero or KataGo, and see if this problem occurs. Furthermore, please specify your game details (komi, handicap stones, etc...) and what is being displayed in the GTP console. Have you opened a file before starting a game?

**Comment by DalurRain** [comment]
Thanks for your reply. And I'm sorry that I opened a new issue rashly. The other issue I refering to is issue#678 and there is no solution for Windows. 

Game details:  komi:6.5, handicap:No stones, Board size: 19x19;
GTP engine settings: just link to the path of PhoenixGo, no args, no init cmds;
I have not opened any file before game;

GTP console shows as follow:
PhoenixGo> name 
= PhoenixGo
PhoenixGo> version 
= 1.0
PhoenixGo> protocol_version 
= 2
PhoenixGo> list_commands 
= name
version
protocol_version
list_commands
quit
clear_board
boardsize
komi
time_settings
time_left
place_free_handicap
set_free_handicap
play
genmove
final_score
get_debug_info
get_last_move_debug_info
use config file 'etc\mcts_cpu.conf' 
log to C:\AppDir\PhoenixGo\log 
start mcts_main 
2020-08-19 16:15:48.607858: I model\zero_model.cc:72] Read checkpoint state succ
2020-08-19 16:15:48.621795: I model\zero_model.cc:80] Read meta graph succ
2020-08-19 16:15:48.622269: I E:\Tensorflow\PhoenixGo\tensorflow\tensorflow\core\platform\cpu_feature_guard.cc:140] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX AVX2
2020-08-19 16:15:48.626325: I model\zero_model.cc:100] Create session succ
2020-08-19 16:15:48.680341: I model\zero_model.cc:107] Create graph succ
2020-08-19 16:15:48.910081: I model\zero_model.cc:119] Load checkpoint succ
PhoenixGo> komi 6.5
? unacceptable komi

game details:
![000](https://user-images.githubusercontent.com/32674546/90610091-89ba7b80-e237-11ea-875e-b07f6acc87e2.jpg)
engine settings:
![001](https://user-images.githubusercontent.com/32674546/90610121-9212b680-e237-11ea-8ca8-dfd6ae10e6e4.jpg)
error and GTP console:
![002](https://user-images.githubusercontent.com/32674546/90610163-9f2fa580-e237-11ea-87ee-b7503481b5ad.jpg)

**Comment by yishn** [comment]
That's the problem:

> PhoenixGo> komi 6.5
> ? unacceptable komi

PhoenixGo apparently doesn't accept 6.5 as a valid komi value. Please change the komi value to something that PhoenixGo accepts.
